tools-cli/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id("com.hedera.hashgraph.java-conventions")
+}
+
+dependencies {
+    implementation(project(":tools-core"))
+    implementation(libs.bundles.logging)
+    implementation(libs.bundles.jackson)
+    implementation(libs.bundles.bouncycastle)
+    implementation(libs.picocli)
+    implementation(libs.eddsa)
+    implementation(libs.grpc.core)
+    implementation(libs.gson)
+//    implementation(libs.json)
```

Comment:
```suggestion
```

Reply:
This reminds me, is there a preference gson/Jackson?